src/virtualenv/create/creator.py

```diff
@@ -131,7 +131,7 @@ def non_write_able(dest, value):
             # pre 3.6 resolve is always strict, aka must exists, sidestep by using os.path operation
             dest = Path(os.path.realpath(raw_value))
         else:
-            dest = value.resolve()
+            dest = value.absolute().resolve()  # on Windows absolute does not imply resolve so use both
```

Comment:
`absolute()` is undocumented (see [bpo-29688](https://bugs.python.org/issue29688))

Reply:
Also relevant: https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573

Reply:
It works in practice though, so I'm fine with this approach for now 🤔

Reply:
ugh, reading up that thread I've swapped to os.path.abspath now 🤷‍♂

Reply:
Also relevant is [bpo-39090](https://bugs.python.org/issue39090) - "getting the absolute path is surprisingly subtle and varied depending on your needs" 🙁